test/config.js

```diff
@@ -279,14 +278,13 @@ test('Config edits trigger re-bundle if using watchify', async (t) => {
   const configPath = './lavamoat/lavamoat-config.json'
   
   bundler.emit('file', './lavamoat/lavamoat-config-override.json')
+  
+  await new Promise(resolve => setTimeout(resolve, 1000))
 
   const configFile = fs.readFileSync(configPath)
   const configFileString = configFile.toString()
-
-  const overrideString = JSON.stringify(configDefault)
-  fs.writeFileSync(overridePath, overrideString)
-
-  await new Promise((resolve) => {
+  
+  await new Promise(async(resolve) => {
```

Comment:
- [ ] Have this promise resolve when update is called, so the rest of those code is not indented
- [ ] use `.once` instead of `.on` since we only care about the first event